backend/src/main/java/com/htecgroup/SkyNest/model/request/UserLoginRequest.java

```diff
@@ -1,8 +1,12 @@
 package com.htecgroup.SkyNest.model.request;
 
 import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 
 @Data
+@Setter
```

Comment:
Doesn't @Data already make all Getters and Setters?

Reply:
I think you're right, I didn't know that.